torch/package/_package_pickler.py

```diff
@@ -113,8 +113,8 @@ def save_global(self, obj, name=None):
                 )
             except UnicodeEncodeError:
                 raise PicklingError(
-                    "can't pickle global identifier '%s.%s' using "
-                    "pickle protocol %i" % (module, name, self.proto)  # type: ignore[attr-defined]
+                    f"can't pickle global identifier '{module}.{name}' using "
+                    f"pickle protocol {self.proto:d}"  # type: ignore[attr-defined]
                 ) from None
```

Comment:
Still don't get why it doesn't raise the cause of the exception of here via from. From None is silly here.

Reply:
Fixed here: https://github.com/pytorch/pytorch/pull/143761